dhtNotWindows.go

```diff
@@ -1,15 +1,15 @@
+//go:build !windows
```

Comment:
I do not think this line is needed, but please educate me if it is.

Reply:
This is needed because it will say to the compiler to not compile this on Windows. Which also is noted in the filename.

Reply:
Doesn't the line below do the same thing?
// +build !windows

Reply:
It wasn't necessary to have both so I removed the +build windows

Reply:
Looks like the syntax has changed. Please leave the old one and add the new one.
Also please add the new format for dhtWindows.go as well.

"Go versions 1.16 and earlier used a different syntax for build constraints, with a "// +build" prefix. The gofmt command will add an equivalent //go:build constraint when encountering the older syntax."
https://pkg.go.dev/cmd/go#hdr-Build_constraints